### DISCUSSION TITLE
Server Memory Requirements

### DISCUSSION THREAD

**Discussion by passing** [jamulussoftware/jamulus#918] [Technical Support & Help]
The [Server Requirements](https://jamulus.io/wiki/Running-a-Server) state:
> Any server should have at least 1.6GHz CPU frequency and 1GB RAM

From my experience with running a server on Linux, you don't need 1GB by far. The server I have setup with max. 20 channels uses just around 40MB. That usage also doesn't go up much when 10 clients are connected.

Having a more accurate value would be especially helpful to clarify that you can also use small cloud instances with less than 1GB. (e.g. a t3.nano from AWS with just 500MB RAM)

**Comment by vdellamea** [comment]
There is a thread running on FB on needed resources, and also on Sourceforge you may find some reports (e.g., https://sourceforge.net/p/llcon/discussion/server/thread/9987543913/#12b2 ). A rough measure someone reported on FB is ~30 clients per core on a i7. On the other side, someone reported jitter with 25 clients on a i5.  Furthermore, mixing depends on N^2, thus computing resources do not grow linearly.
When it comes to cloud servers, I personally found that, with 7 clients, t3.micro has more artifacts than c5.large (server side, on the recording of all the tracks). I do not think it's matter of memory in particular, but the cumulated effect of limitations on smaller cloud machines (burstable CPU and network, shared physical resource, etc) not always make it possible to have sustained good results. 
Thus, while it is true that a smaller machine could be used in certain cases, I would not push towards the bottom, to avoid having users unstatisfied  at their first approach. Better to give a safe configuration, suggesting to try a downgrade after having set everything.

**Comment by ann0see** [comment]
Once this gets resolved, please comment your findings here: https://github.com/jamulussoftware/jamuluswebsite/issues/54

**Comment by ann0see** [comment]
@mulyaj I think this might be interesting for you concerning the new getting started page.
